spec/bundler/shared_helpers_spec.rb

```diff
@@ -279,35 +279,37 @@
       )
     end
 
-    it "does not exit if bundle path is the jruby/warbler standard uri path", :ruby => "> 1.8.7" do
-      allow(Gem).to receive(:path_separator).and_return(
-        /(?<!jar:file|jar|file|classpath|uri:classloader|uri|http|https):/
-      )
-      allow(Bundler).to receive(:bundle_path) { Pathname.new("uri:classloader:/WEB-INF/gems") }
-      expect { subject.send(:validate_bundle_path) }.not_to raise_error(Bundler::PathError)
-    end
-
-    it "exits if bundle path contains another directory with the jruby uri path separator", :ruby => "> 1.8.7" do
-      allow(Gem).to receive(:path_separator).and_return(
-        /(?<!jar:file|jar|file|classpath|uri:classloader|uri|http|https):/
-      )
-      allow(Bundler).to receive(:bundle_path) {
-        Pathname.new("uri:classloader:/WEB-INF/gems:other/dir")
-      }
+    if RUBY_VERSION.to_f > 1.8 # In versions of jruby that supported ruby 1.8, the path separator was the standard File::PATH_SEPARATOR
```

Comment:
please use the `:ruby => "1.9"` tag on the specs that require this

Reply:
After many contortions, this is now done.